server/src/main/java/org/opensearch/identity/tokens/BearerAuthToken.java

```diff
@@ -58,4 +58,9 @@ public String getTokenIdentifier() {
     public String toString() {
         return "Bearer auth token with header=" + header + ", payload=" + payload + ", signature=" + signature;
     }
+
+    @Override
+    public String getTokenValue() {
```

Comment:
Hi @cwperks. According to the work we are doing in the security plugin, should we keep this? Or we need to add 2 more interfaces of `issueOnBehalfOfToken` and `issueServiceAccountToken`?